### DISCUSSION TITLE
Welcome to git-DAO Discussions!

### DISCUSSION THREAD

**Discussion by carlbarrdahl** [carlbarrdahl/git-dao#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sudoFerraz** [comment]
Hey @carlbarrdahl, I think you will like what we are building atm bc of gitDAO, I followed you on twitter, let's connect and have a chat :)
